dataactcore/migrations/versions/827e8db6242e_.py

```diff
@@ -0,0 +1,36 @@
+""" Merging duns historic flag and princple place street
```

Comment:
Please see other merge migrations for examples of what it should look like. The message should say which hashes are being merged and the name of the file should reflect this as well